package-lock.json

```diff
@@ -0,0 +1,6 @@
+{
+  "name": "Budget-planner",
```

Comment:
Jak dla mnie jedna kopia plików `package.json` i `package-lock.json` wystarczy 😏 Skoro mamy już oddzielnie frontend i backend, to moim zdaniem jsony powinny się znajdować w projekcie frontu, a wyżej już nie ma potrzeby.

Reply:
Sorry, zapomniałem usunąć - to jakiś artefakt po instalacji package'y w złym folderze przez przypadek